requirements.txt

```diff
@@ -1,5 +1,6 @@
 cyvcf2
 dask[array]
+fsspec
```

Comment:
I worry a little bit about dependency sprawl here, but it seems like dask already depends on fsspec, so it makes no difference in practise?

Reply:
Yes, I don't think there's any way round this. The cloud implementations of fsspec (e.g. gcfs) are not dependencies - the user has to add them as needed.